examples/ssh/backup_ftps.js

```diff
@@ -0,0 +1,65 @@
+/**
```

Comment:
The name of the file is incorrect.

Can you change to something more meaningful

sftp_file